content/_index.md

```diff
@@ -9,50 +9,37 @@ by `GoldCard holders` for `GoldCard holders`
 
 <br>
 
-
-
-
-
 ## <i class="uil uil-question-circle"></i> Applying for the Gold Card?
-<div class="list-q">
-- All the answers about the Gold Card<br>
-- How long does the process take?<br>
-- What documents do I need to provide?<br>
-- Do I qualify for the Gold card?<br>
 
-</div>
+- All the answers about the Gold Card
+- How long does the process take?
+- What documents do I need to provide?
+- Do I qualify for the Gold card?
 
-{{< button "application-faq/" "Access the FAQ" >}} 
+{{< button "application-faq/" "Access the FAQ" >}}
 
 <br>
 <br>
 
-
 ## <i class="uil uil-trophy"></i> Already a Gold Card Holder?
-<div class="list-q">
-- Access to our chat group where we help each other<br>
-- Jobs opportunities that require our expertise<br>
-- Visibility if you are looking for new opportunities<br>
-- Receive a regular newsletter with the latest GoldCard News<br>
+
+- Access to our chat group where we help each other
+- Jobs opportunities that require our expertise
+- Visibility if you are looking for new opportunities
+- Receive a regular newsletter with the latest GoldCard News
 
 {{< button "https://forms.gle/K88uVy2jMW61DpT2A" "Join the Gold Card Network" >}}
 {{< button "goldcard-holders-faq/" "Resources for GoldCard holders" >}}
 
-</div>
-
 <br>
 <br>
 
-
-
 ## <i class="uil uil-newspaper"></i> News of the Gold Card Network
-- [2020-06-05] [Goldcard Holder: Tomáš Řízek illustration exhibition](https://99dac.com/exhibition-detail.php?id=140)
-- [2020-05-27] [GoldCard Holder: Starting an Eye Yoga Career](https://meet.bnext.com.tw/intl/articles/view/46488)
-- [2020-04-30] [More than 700 Gold Cards Issued!](https://foreigntalentact.ndc.gov.tw/en/News_Content.aspx?n=F0746484B877D582&s=91B121FE3FA7C24D)
-- [2020-04-13] [Tax payment has been extended to June 30th 2020](https://home.kpmg/us/en/home/insights/2020/04/tnf-taiwan-tax-return-tax-payment-deadlines-extended-covid-19.html)
-
-
 
+- [2020-06-05][goldcard holder: tomáš řízek illustration exhibition](https://99dac.com/exhibition-detail.php?id=140)
```

Comment:
This results in no space between the date tag and the label. Does that look OK? It seemed a bit odd to me at first glance.

Reply:
Oh interesting, this was accidental. I have an auto-formatter for markdown and I'm guessing this was a result of that auto formatting